src/Scout/ScoutEnhancer.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace Nuwave\Lighthouse\Scout;
+
+use Illuminate\Database\Eloquent\Builder as EloquentBuilder;
+use Laravel\Scout\Builder as ScoutBuilder;
+use Laravel\Scout\Searchable;
+use Nuwave\Lighthouse\Execution\Arguments\Argument;
+use Nuwave\Lighthouse\Execution\Arguments\ArgumentSet;
+use Nuwave\Lighthouse\Schema\Directives\SearchDirective;
+use Nuwave\Lighthouse\Support\Contracts\ArgBuilderDirective;
+use Nuwave\Lighthouse\Support\Utils;
+
+class ScoutEnhancer
+{
+    /**
+     * @var ArgumentSet
+     */
+    protected $argumentSet;
+
+    /**
+     * @var object
```

Comment:
We should be able to change this to the Scout builder type right?

Reply:
This one is actually the non-scout builder